README.md

```diff
@@ -23,7 +23,7 @@ STRIPE_KEY=STRIPE_KEY
 
 # The ID of the email price product used in billing subscribers.
 EMAIL_PRODUCT_PRICE_ID=EMAIL_PRODUCT_PRICE_ID
```

Comment:
@vickywane is this doc line still necessary? We do not hardcode product ID in the API.

Reply:
For now, yes it is. 

The `EMAIL_PRODUCT_PRICE_ID ` that is used to retrieve only the pricing entity is different from the `EMAIL_PRODUCT_ID`. 

The `EMAIL_PRODUCT_PRICE_ID` is used in the `/subscription` function when creating a subscription.  The product ID response retrieved using the `/product` function does not include the ID of the pricing entity. 

See [subscription](https://stripe.com/docs/api/subscriptions/create?lang=node) and [price](https://stripe.com/docs/api/prices/create) docs.

I would further rephrase the description of the `EMAIL_PRODUCT_PRICE_ID` to better reflect it's use